docs/architecture/grpc-for-wcf-developers/load-balancing.md

```diff
@@ -1,43 +1,43 @@
 
-An example of an L4 load balancer is the [Azure Load Balancer](https://azure.microsoft.com/services/load-balancer/).
+ [Azure Load Balancer](https://azure.microsoft.com/services/load-balancer/) is an example of an L4 load balancer.
 
 ## L7 load balancers
 
 An L7 load balancer parses incoming HTTP/2 requests and passes them on to back-end instances on a request-by-request basis, no matter how long the connection is held by the client.
 
-Examples of L7 load balancers include:
+Examples of L7 load balancers:
 
-- [Nginx](https://www.nginx.com/)
-- [HAproxy](https://www.haproxy.com/)
+- [NGINX](https://www.nginx.com/)
+- [HAProxy](https://www.haproxy.com/)
 - [Traefik](https://traefik.io/)
 
-As a rule of thumb, L7 load balancers are the best choice for gRPC and other HTTP/2 applications (and for HTTP applications generally, in fact). L4 load balancers will *work* with gRPC applications, but are primarily useful when low latency and low overhead are of paramount importance.
+As a rule of thumb, L7 load balancers are the best choice for gRPC and other HTTP/2 applications (and for HTTP applications generally, in fact). L4 load balancers will *work* with gRPC applications, but they're mainly useful when low latency and low overhead are important.
 
 > [!IMPORTANT]
-> At the time of this writing, not all L7 load balancers support all parts of the HTTP/2 specification required by gRPC services, such as trailing headers.
+> At the time of this writing, some L7 load balancers don't support all the parts of the HTTP/2 specification that are required by gRPC services, such as trailing headers.
 
-When using TLS encryption, load balancers can terminate the TLS connection and pass unencrypted requests to the back-end application, or pass the encrypted request along. Either way, the load balancer will need to be configured with the server's public and private key, so that it can decrypt requests for processing.
+If you're using TLS encryption, load balancers can terminate the TLS connection and pass unencrypted requests to the back-end application, or they can pass the encrypted request along. Either way, the load balancer will need to be configured with the server's public and private key so it can decrypt requests for processing.
 
-Refer to the documentation for your preferred load balancer to find out how to configure it to handle HTTP/2 requests with your back-end services.
+See to the documentation for your preferred load balancer to find out how to configure it to handle HTTP/2 requests with your back-end services.
 
 ## Load balancing within Kubernetes
 
-See [the section on Service Meshes](service-mesh.md) for a discussion of load balancing across internal services on Kubernetes.
+See [the section on service meshes](service-mesh.md) for a discussion of load balancing across internal services on Kubernetes.
```

Comment:
Seems inconsistent to me. Why are we capitalizing Pods and Services, but not Service Meshes?

Reply:
Capitalization of Pod and Service looked inconsistent to me before the edit. I decided to capitalize them consistently because they're objects. I'm not sure if that's the best way to go and am open to suggestions. Service mesh isn't capitalized in the article about them.

Reply:
I figured I'd take a look at the k8s docs to see how they refer to pods: https://kubernetes.io/docs/concepts/workloads/pods/pod/#uses-of-pods

Looks like they've capitalized it as well.

Reply:
It's funny that even their doc is not consistent 😄